### PR TITLE
update copy on r/chart-studio docs 

### PR DESCRIPTION
**Pull request by jdamiba**
The purpose of this PR is to clarify that the docs in the `r/chart-studio` section refer to using the `plotly` package with Chart Studio.

### CODE REVIEW

File: _posts/r/chart-studio/2015-04-09-static-image_r_index.Rmd [hunk 1]
```diff
@@ -38,24 +38,29 @@ library(plotly)
...
 ```
 
-### Supported Formats
+### Supported Formats For Static Image Export
 
-The common image formats: 'PNG', 'JPG/JPEG' are supported. In addition, formats like 'EPS', 'SVG' and 'PDF' are also available for user with a Personal or Professional subscription. You can get more details on our [pricing page] (https://plot.ly/products/cloud/)
+With the `plotly` R package, you can export your charts as static images in the `.png` and `.jpg/.jpeg` file formats.
 
-**Note:** It is important to note that any figures containing WebGL traces (i.e. of type scattergl, heatmapgl, contourgl, scatter3d, surface, mesh3d, scatterpolargl, cone, streamtube, splom, or parcoords) that are exported in a vector format like SVG, EPS or PDF will include encapsulated rasters instead of vectors for some parts of the image.
+Currenlty, exporting charts as static images in the `.eps`, `.svg`, and `.pdf` formats is a feature that is available only to users of [Chart Studio Enterprise](https://plot.ly/online-chart-maker/).
```
Comment (nicolaskruchten) [inline _posts/r/chart-studio/2015-04-09-static-image_r_index.Rmd]:
```suggestion
Currently, exporting charts as static images in the `.eps`, `.svg`, and `.pdf` formats is a feature that is available only to users of [Chart Studio Enterprise](https://plot.ly/online-chart-maker/).
```

File: _posts/r/chart-studio/2015-04-09-static-image_r_index.Rmd [hunk 2]
```diff
@@ -65,19 +70,24 @@ p <- plot_ly(z = ~volcano) %>% add_surface()
 orca(p, "surface-plot.svg")
 ```
 
-### Export Using Your Plotly Account
+#### Using Chart Studio to Export Static Images
 
-Another option is to do image export through your plotly account.
+Another option for exporting the R charts you create with the `plotly` R package is to use your Chart Studio account.
```
Comment (nicolaskruchten) [inline _posts/r/chart-studio/2015-04-09-static-image_r_index.Rmd]: Link to the "chart studio getting started" page from "your Chart Studio account" ?

File: _posts/r/chart-studio/2015-04-09-static-image_r_index.Rmd [hunk 3]
```diff
@@ -1,10 +1,10 @@
 ---
-description: How to export plotly graphs as static images in R. Plotly supports png,
-  svg, jpg, and pdf image export.
+description: How to export R graphs as static images. Ploty's R graphing library supports exporting to .png,
```
Comment (nicolaskruchten) [inline _posts/r/chart-studio/2015-04-09-static-image_r_index.Rmd]: High level comment here... this page should actually be split into two pages: one in Plotly Fundamentals that covers just Orca and does not link to the Chart Studio page, and one in Chart Studio that does link to the Orca page saying "Chart Studio is an alternative"

File: _posts/r/chart-studio/2015-04-09-static-image_r_index.Rmd [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
+  .svg, .jpg, and .pdf file formarts and exporting Chart Studio graphs.
 display_as: chart_studio
 language: r
 layout: base
-name: Static Image Export
+name: Using Chart Studio to Export Static Images
```
[the remark this answers is not in the document]
Comment (nicolaskruchten) [reply _posts/r/chart-studio/2015-04-09-static-image_r_index.Rmd]: never mind, see comment about splitting this page.

[the remark this answers is not in the document]
Comment (nicolaskruchten) [reply _posts/r/chart-studio/2015-04-09-static-image_r_index.Rmd]: never mind, see comment about splitting this page.

File: _posts/r/chart-studio/2020-01-17-getting-started-with-chart-studio.Rmd [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+```{r, echo = FALSE, message=FALSE}
+knitr::opts_chunk$set(message = FALSE, warning=FALSE)
+```
+
+# Getting Started with Chart Studio and the `plotly` R Package
+
+`plotly` is an R package for creating interactive web-based graphs via the open source JavaScript graphing library [plotly.js](http://plot.ly/javascript). 
+
+As of version 2.0 (November 17, 2015), R graphs created with the `plotly` R package are rendered *locally* through the [htmlwidgets](http://www.htmlwidgets.org/) framework.
```
Comment (nicolaskruchten) [inline _posts/r/chart-studio/2020-01-17-getting-started-with-chart-studio.Rmd]:
```suggestion
As of version 2.0 (November 17, 2015), R graphs created with the `plotly` R package are, by default, rendered *locally* through the [htmlwidgets](http://www.htmlwidgets.org/) framework.
```

File: _posts/r/chart-studio/2015-04-09-static-image_r_index.Rmd [hunk 6]
```diff
@@ -87,7 +83,12 @@ plotly_IMAGE(p, format = "png", out_file = "output.png")
 
 ![](https://images.plot.ly/plotly-documentation/images/output.png)
 
-### Appending File Type to URL
+### Alternative Methods Of Exporting Graphs As Static Images In R
+
+#### Local Image Export
+
+As an alternative to using the Chart Studio web service to export your R graphs as static images, you can [use the built-in `orca()` function](https://plot.ly/r/image-export-orca) to export images locally.
```
Comment (nicolaskruchten) [inline _posts/r/chart-studio/2015-04-09-static-image_r_index.Rmd]: see comment in the other PR about permalinks.

File: _posts/r/chart-studio/2015-04-09-static-image_r_index.Rmd [hunk 7]
```diff
@@ -1,16 +1,16 @@
...
-name: Static Image Export
+name: Exporting Graphs As Static Images Using Chart Studio
 order: 2
 output:
   html_document:
     keep_md: true
 page_type: example_index
-permalink: r/static-image-export/
+permalink: r/chart-studio-image-export/
+redirect_from: r/static-image-export/
```
Comment (nicolaskruchten) [inline _posts/r/chart-studio/2015-04-09-static-image_r_index.Rmd]: I don't think we should `redirect_from` here. This permalink should belong to the Orca page.

File: _posts/r/chart-studio/2015-07-30-get-requests.Rmd [hunk 8]
```diff
@@ -1,15 +1,17 @@
...
 layout: base
-name: Get Requests
+name: Working With Chart Studio Graphs
 order: 6
 output:
   html_document:
     keep_md: true
-permalink: r/get-requests/
-thumbnail: thumbnail/get-requests.jpg
+permalink: r/working-with-chart-studio-graphs/
```
Comment (nicolaskruchten) [inline _posts/r/chart-studio/2015-07-30-get-requests.Rmd]: consider making the same URL and title change in the Python equivalent file in this same repo

File: _posts/r/chart-studio/2015-08-10-knitr.Rmd [hunk 9]
```diff
@@ -1,19 +1,18 @@
 ---
-description: How to embed Plotly graphs in Rmd files.
+description: How to embed R graphs in RMarkdown files.
 display_as: chart_studio
 language: r
 layout: base
-name: Embedding Graphs with Knitr
+name: Embedding R Graphs in RMarkdown Files
```
Comment (nicolaskruchten) [inline _posts/r/chart-studio/2015-08-10-knitr.Rmd]: the "R" here will look weird when we add "in R" automatically

File: _posts/r/chart-studio/2015-07-30-get-requests.Rmd (hunk 8 above)
Comment (jdamiba) [reply _posts/r/chart-studio/2015-07-30-get-requests.Rmd]: Changed the Python equivalent doc title and URL, and added a `redirect_from`.

File: _posts/r/chart-studio/2015-08-10-knitr.Rmd (hunk 9 above)
Comment (jdamiba) [reply _posts/r/chart-studio/2015-08-10-knitr.Rmd]: Removed the extraneous R.

File: _posts/r/chart-studio/2015-07-30-get-requests.Rmd (hunk 8 above)
Comment (nicolaskruchten) [reply _posts/r/chart-studio/2015-07-30-get-requests.Rmd]: I can't see that change... did you commit it?

**Comment by nicolaskruchten** [comment]
Let's remove all the "New to Plotly" and "Version Check" sections from these R docs too please, to match https://github.com/plotly/plotly.r-docs/pull/12

File: _posts/r/chart-studio/2015-07-30-get-requests.Rmd (hunk 8 above)
Comment (jdamiba) [reply _posts/r/chart-studio/2015-07-30-get-requests.Rmd]: Yes, I committed the file -> https://github.com/plotly/documentation/blob/53d50751a724448754079873f1cb21302de32f0e/_posts/python/chart-studio/2019-07-03-get-requests.html

**Comment by jdamiba** [comment]
@nicolaskruchten This PR has been updated and is ready for review. I have removed the `New To Plotly` and `Version Check` sections from these files.

**Comment by nicolaskruchten** [comment]
💃

**Pull request merged**